assets/app/views/services.html

```diff
@@ -16,7 +16,12 @@
         <div>Container port: {{service.spec.containerPort}}</div>
         <div>Protocol: {{service.spec.protocol}}</div>
         <div>Selectors:<span ng-repeat="(selectorLabel, selectorValue) in service.spec.selector"> {{selectorLabel}}={{selectorValue}}</span></div>
```

Comment:
@jwforres do we want comma-separation here as well? (e.g. `<span ng-show="!$last">, </span>`)